management/organization.go

```diff
@@ -393,12 +405,12 @@ func (m *OrganizationManager) AssociateClientGrant(ctx context.Context, id strin
 	}{
 		GrantID: grantID,
 	}
-	err = m.management.Request(ctx, "POST", m.management.URI("organizations", id, "client-grants"), &body, applyListDefaults(opts))
```

Comment:
Why is `applyListDefaults` incorrect here?

Reply:
As this is a POST to associate a client grant and organization it doesn't support the parameters that `applyListDefaults` sets (such as `per_page` and `include_totals`)

Reply:
Thanks for the clarification 👍🏼